beetmoverscript/script.py

```diff
@@ -12,16 +12,16 @@
 import aiohttp
```

Comment:
out of the three current scriptworker implementations, balrogscript is the odd one out:

https://github.com/mozilla-releng/beetmoverscript/tree/master/beetmoverscript
https://github.com/mozilla-releng/signingscript/tree/master/signingscript
https://github.com/mozilla-releng/balrogscript/tree/master/balrogscript

I agree consistency would be better. Happy to go with {scriptworker_type}script.py naming convention if that's the way we want it.